marker/v2/builders/layout.py

```diff
@@ -49,7 +49,7 @@ def add_blocks_to_pages(self, pages: List[PageGroup], layout_results: List[Layou
             layout_page_size = PolygonBox.from_bbox(layout_result.image_bbox).size
             provider_page_size = page.polygon.size
             for bbox in sorted(layout_result.bboxes, key=lambda x: x.position):
-                block_cls = LAYOUT_BLOCK_REGISTRY[BlockTypes[bbox.label]]
+                block_cls = BLOCK_REGISTRY[BlockTypes[bbox.label]]
```

Comment:
Should we just always call something like `get_block_cls` versus directly indexing?  I don't mind either way, but I think it should be consistent.

Reply:
yeah you're right! I fixed it